models/fivetran_platform__connector_daily_events.sql

```diff
@@ -7,39 +7,56 @@ with connector as (
 ),
 
 -- grab api calls, schema changes, and record modifications
+
 log_events as (
 
     select 
         connector_id,
         cast( {{ dbt.date_trunc('day', 'created_at') }} as date) as date_day,
-        case 
-            when event_subtype in ('create_table', 'alter_table', 'create_schema', 'change_schema_config') then 'schema_change' 
-            else event_subtype end as event_subtype,
-
-        sum(case when event_subtype = 'records_modified' then cast( {{ fivetran_log.fivetran_log_json_parse(string='message_data', string_path=['count']) }} as {{ dbt.type_bigint()}} )
-        else 1 end) as count_events 
+        event_subtype,
+        message_data
 
     from {{ ref('stg_fivetran_platform__log') }}
 
-    where event_subtype in ('api_call', 
-                            'records_modified', 
-                            'create_table', 'alter_table', 'create_schema', 'change_schema_config') -- all schema changes
-                            
+    where event_subtype in (
+        'api_call', 'extract_summary', 'records_modified', 'create_table', 'alter_table',
+        'create_schema', 'change_schema_config') -- all schema changes
```

Comment:
Small note that I know isn't related to your changes directly, but is it accurate to have the comment `all schema changes` in this filter? For example, `api_call` and `extract_summary` are not inherently schema changes and we make this clear in the below `case when` statement where we categorize perform the following:
```sql
when event_subtype in ('create_table', 'alter_table', 'create_schema', 'change_schema_config') then 'schema_change'
```

If this is true, can we change this comment to be `all relevant event subtypes` or something along those lines.

Reply:
Makes sense--updated!